### PR TITLE
[LPN-1334-BO] add department site in the BO and the view page

### PR DESCRIPTION
**Pull request by elvino29**

### CODE REVIEW

File: src/Admin/DepartmentSite/DepartmentSiteAdmin.php [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        ;
+    }
+
+    protected function configureFormFields(FormMapper $formMapper): void
+    {
+        $formMapper
+            ->with('Département', ['class' => 'col-md-6'])
+                ->add('zone', ModelAutocompleteType::class, [
+                    'property' => ['name', 'code'],
+                    'label' => 'Zone départementale',
```
Comment (ottaviano) [inline src/Admin/DepartmentSite/DepartmentSiteAdmin.php:96]:
```suggestion
                    'label' => 'Département',
```

File: src/Controller/Renaissance/DepartmentSite/DepartmentSiteController.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+/**
+ * @Route("/federations", name="app_renaissance_department_site_")
+ */
+class DepartmentSiteController extends AbstractController
+{
+    /**
+     * @Route("/{slug}", name="view"), methods={"GET"}
+     * @Entity("departmentSite", expr="repository.findOneBySlug(slug)")
+     */
+    public function departmentSiteAction(DepartmentSite $departmentSite): Response
```
Comment (ottaviano) [inline src/Controller/Renaissance/DepartmentSite/DepartmentSiteController.php:20]: what about making this controller invokable, with `__invoke` method?

File: src/Entity/DepartmentSite/DepartmentSite.php [hunk 3]
```diff
@@ -91,7 +91,7 @@ class DepartmentSite
      * @ORM\Column(unique=true)
      * @Gedmo\Slug(handlers={
      *     @Gedmo\SlugHandler(class="App\DepartmentSite\DepartmentSiteSlugHandler")
-     * }, fields={"slug"}, updatable=true)
+     * }, fields={"content"})
```
Comment (ottaviano) [inline src/Entity/DepartmentSite/DepartmentSite.php:94]: why do you change this for content field?

File: src/Controller/Renaissance/DepartmentSite/DepartmentSiteController.php (hunk 2 above)
Comment (elvino29) [reply src/Controller/Renaissance/DepartmentSite/DepartmentSiteController.php:20]: I will add the list action in this file on another PR

**Review by ottaviano — APPROVED** [review]

**Pull request merged**